app.css

```diff
@@ -0,0 +1,21 @@
+* {
+  box-sizing: border-box;
+
+}
```

Comment:
Good work with the box-sizing reset.

Also remember to reset `body` `margin` so that elements are full width on the page and this will help t remove the white border around the full page.